symengine/fields.h

```diff
@@ -91,8 +91,10 @@ class GaloisFieldDict
     // Returns whether polynomial is squarefield in `modulo_`
     bool gf_is_sqf() const;
 
-    // Returns the square free decomposition of polynomial's monic representation in `modulo_`
-    // A vector of pair is returned where each element is a factor and each pair's first
+    // Returns the square free decomposition of polynomial's monic
+    // representation in `modulo_`
+    // A vector of pair is returned where each element is a factor and each
+    // pair's first
     // raised to power of second gives the factor.
```

Comment:
Could you change this to 
`// pair's first raised to power of second gives the factor.`
Looks cleaner.